.github/workflows/closed-fixed-pending-release.yml

```diff
@@ -0,0 +1,14 @@
+name: Closed fixed-pending-release issues
+on:
+    release:
+        types: [published]
+
+jobs:
+    close_fixed_pending_release:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Close issues marked 'fixed-pending-release'
+              uses: gcampbell-msft/fixed-pending-release@0.0.12
```

Comment:
Is it possible to host the action directly in this repository, or move it to the `microsoft` org? I'd like to avoid permissions issues if we need to modify this action in the future.

Reply:
You mean the actual GitHub Action that's published to the marketplace? 

I don't think we'd want to host it directly in this repository, since it's possible for it to be used in other repositories such as the makefile extension. What is the process / requirements for moving it to the `microsoft` org?

Reply:
You'd need to follow [this OSS release process](https://docs.opensource.microsoft.com/releasing/). If the code is small enough, it will likely qualify for expedited approval, or no formal approval at all.

Reply:
For now, we'll leave it as is. Once we confirm that it suits our needs and works well, we'll investigate this. Thanks.